### PR TITLE
Add mysql CI host

### PR DESCRIPTION
**Pull request by cgole**

### CODE REVIEW

File: test/init.js [hunk 1]
```diff
@@ -7,8 +7,8 @@ var config = require('rc')('loopback', {test: {mysql: {}}}).test.mysql;
 global.getConfig = function (options) {
 
   var dbConf = {
-    host: config.host || 'localhost',
-    port: config.port || 3306,
+    host: process.env.MYSQL_HOST || config.host || 'localhost',
+    port: process.env.MYSQL_PORT || config.port || 3306,
     database: 'myapp_test',
```
Comment (cgole) [inline test/init.js]: @rmg the database name is used by the tests so we'll just have to run those builds sequentially if they conflict. I'll add the username and password

**Comment by rmg** [comment]
Would be nice if the tests were passing, but I'm guessing they were failing before this PR:
```
migrations
      ․ should run migration: 78ms
      ․ UserData should have correct columns: 5ms
      1) UserData should have correct indexes
      ․ StringData should have correct columns: 6ms
      ․ NumberData should have correct columns: 4ms
      ․ DateData should have correct columns: 3ms
      ․ should autoupdate: 117ms
      ․ should check actuality of dataSource: 24ms
      ․ should allow numbers with decimals: 23ms
      ․ should allow both kinds of date columns: 15ms
      ․ should report errors for automigrate: 0ms
      ․ should report errors for autoupdate: 0ms
      ․ should disconnect when done: 0ms
```

LGTM

File: test/init.js [hunk 2]
```diff
@@ -3,15 +3,15 @@ module.exports = require('should');
...
-    username: config.username,
-    password: config.password,
+    username: process.env.MYSQL_USERNAME || config.username,
```
Comment (rmg) [inline test/init.js]: Should be `MYSQL_USER`, not `MYSQL_USERNAME`.

Comment (cgole) [inline test/init.js]: I can make that change, but may I ask why?

Comment (rmg) [inline test/init.js]: Because that is what is being set by CI and used by other repos that look for it.

**Comment by cgole** [comment]
test please

File: test/init.js [hunk 3]
```diff
@@ -3,15 +3,15 @@ module.exports = require('should');
 var DataSource = require('loopback-datasource-juggler').DataSource;
 
 var config = require('rc')('loopback', {test: {mysql: {}}}).test.mysql;
-
+console.log(config)
```
Comment (rmg) [inline test/init.js]: This should be removed before merging.

File: test/init.js [hunk 4]
```diff
@@ -20,7 +20,7 @@ global.getConfig = function (options) {
       dbConf[el] = options[el];
     }
   }
-
+  console.log(dbConf)
```
Comment (rmg) [inline test/init.js]: This should be removed before merging.

**Comment by rmg** [comment]
Other than the `console.log()` lines, LGTM.

**Comment by cgole** [comment]
test please

**Comment by cgole** [comment]
test please

**Comment by rmg** [comment]
@cgole since the changes in this PR are only in the tests, you should be able to ignore any downstream/dependant build failures as they would be unrelated.

**Pull request merged**